graylog2-server/src/main/java/org/graylog2/indexer/indexset/IndexSetConfig.java

```diff
@@ -53,7 +55,7 @@
 @JsonAutoDetect
 @DbEntity(collection = MongoIndexSetService.COLLECTION_NAME,
           readPermission = INDEXSETS_READ)
-public abstract class IndexSetConfig implements Comparable<IndexSetConfig>, SimpleIndexSetConfig {
+public abstract class IndexSetConfig extends ScopedEntity implements Comparable<IndexSetConfig>, SimpleIndexSetConfig {
```

Comment:
This is a scoped entity now, but the service doesn't treat it as one. Should we maybe put both (making it a scoped entity and having the service respect the scope) into a separate PR? If we are sure that a follow-up PR is gonna get in timely, then I can live with having this split across PRs, but otherwise I think it will be confusing that a scoped entity's scope is ignored.

Reply:
Scoping is what I originally set out to do: Graylog2/graylog-plugin-enterprise#5904. The migration is an afterthought.